source/addComment.js

```diff
@@ -1,3 +1,13 @@
 $(document).ready(function(){
-  // add your code here
+  var commentForm = $(".comment-form");
+
+  $("#new_comment_button").on("click", function(event){
+    event.preventDefault();
+    commentBody = $("#comment-body").val()
+    author = $("#author-name").val()
+    $('#comment_list').append('<li>' + commentBody + "<span class='author'>" + author + "</span></li>");
+    $("#comment-body").val('');
+    $("#author-name").val('');
```

Comment:
there's a lot going on in this callback function. it'd be pretty easy to pull some of this out... for example,

```js
function clearValues() {
  $("#comment-body").val('')
  $("#author-name").val('')
}
```

then we can replace these two lines with `clearValues()`. same goes with constructing the html 👍